### ISSUE TITLE
Your project has pages but the `<NuxtPage />` component has not been used

### ISSUE THREAD

**Issue by OrbisK** [antfu/node-modules-inspector#8]
### Describe the bug

1. `pnpm run dev`
2. open localhost
3.  see console

![Image](https://github.com/user-attachments/assets/a9946f11-4a52-40cd-ada3-6f6553730790)

> [nuxt] Your project has pages but the `<NuxtPage />` component has not been used. You might be using the `<RouterView />` component instead, which will not work correctly in Nuxt. You can set `pages: false` in `nuxt.config` if you do not wish to use the Nuxt `vue-router` integration.

`<NuxtPage>` is used, but not detected? 

### Reproduction

http://localhost:3000/grid

### System Info

```Shell
System:
    OS: Linux 6.8 Ubuntu 24.04.1 LTS 24.04.1 LTS (Noble Numbat)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 46.67 GB / 62.58 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 23.5.0 - ~/.nvm/versions/node/v23.5.0/bin/node
    npm: 10.9.2 - ~/.nvm/versions/node/v23.5.0/bin/npm
    pnpm: 9.15.4 - ~/.nvm/versions/node/v23.5.0/bin/pnpm
    bun: 1.1.42 - ~/.nvm/versions/node/v23.5.0/bin/bun
  Browsers:
    Chrome: 130.0.6723.69
```

### Used Package Manager

pnpm

### Validations

- [x] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [x] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [x] I am willing to submit a PR to fix this issue
- [x] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by antfu** [comment]
It's a bug (false-positive) of Nuxt when the project is not starting from the root. I plan to have a look but haven't get the time to

**Comment by antfu** [comment]
Let's just ignore that for now.
